### PR TITLE
Fix/kryo issue

### PR DESCRIPTION
**Pull request by dmitriyisaev**

### CODE REVIEW

File: conf/serialization.conf [hunk 1]
```diff
@@ -144,6 +146,7 @@ akka.actor {
     "models.rpc.PongMessage" = kryo
     "models.rpc.ResponseMessage" = kryo
     "models.rpc.RequestMessage" = kryo
+    "models.rpc.DSAError" = kryo
```
Comment (ilya-martynov) [inline conf/serialization.conf]: I believe this is redundant. That error we saw in the log says about Kryo mapping only, means it is already bind.

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Review by ilya-martynov — APPROVED** [review]

**Comment by ilya-dubovskiy** [comment]
Do not merge!!

**Comment by ilya-dubovskiy** [comment]
Build and tests are GOOD

**Pull request merged**